libs/partners/box/pyproject.toml

```diff
@@ -0,0 +1,97 @@
+[tool.poetry]
+name = "langchain-box"
+version = "0.1.0"
+description = "An integration package connecting Box and LangChain"
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain/tree/master/libs/partners/box"
+
+[tool.poetry.dependencies]
```

Comment:
in the end, ideally dependency list should be `python` `langchain-core` and `box-sdk-gen`